### PR TITLE
new test Veth Dlpar.

### PR DESCRIPTION
**Pull request by smruti77**
new test for veth dlpar operation.

Signed-off-by: bismurti bidhibrata pattajoshi <bbidhibr@in.ibm.com>

### CODE REVIEW

**Comment by smruti77** [comment]
[veth_dlpar.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/4481389/veth_dlpar.log)

File: io/net/virt-net/veth_dlpar.py [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        cmd = "lsmap -all -net"
+        output = self.run_command(cmd)
+        for line in output.splitlines():
+            if self.slot in line:
+                self.iface = line.split()[0]
+        cmd = "lsmap -vadapter %s -net" % self.iface
+        output = self.run_command(cmd)
+        for line in output.splitlines():
+            if "SEA" in line:
+                self.sea = line.split()[-1]
```
Comment (narasimhan-v) [inline io/net/virt-net/veth_dlpar.py:79]: cancel when `self.sea` is not got.

File: io/net/virt-net/veth_dlpar.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        self.log.info(self.sea)
+        local = LocalHost()
+        self.networkinterface = NetworkInterface(self.interface, local)
+        if self.networkinterface.ping_check(self.peer_ip, count=5) is not None:
+            self.fail("peer connection is failed")
```
Comment (narasimhan-v) [inline io/net/virt-net/veth_dlpar.py:84]: self.cancel would be better, IMO

File: io/net/virt-net/veth_dlpar.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        '''
+        cmd = "mkdev -l %s" % self.sea
+        cmd_l = "echo \"%s\" | oem_setup_env" % cmd
+        try:
+            output = self.run_command(cmd_l)
+            self.log.info(output)
+        except CommandFailed as cf:
+            self.fail("Failed dlpar add operation as %s" % cf)
+        if self.networkinterface.ping_check(self.peer_ip, count=5) is not None:
+            self.fail("ping failed after add operation")
```
Comment (narasimhan-v) [inline io/net/virt-net/veth_dlpar.py:151]: This should be done in the `test_dlpar()` or as a separate function, not inside `veth_dlpar_add()`

Comment (smruti77) [reply io/net/virt-net/veth_dlpar.py:151]: ok

**Pull request merged**